dockstore-client/src/main/java/io/dockstore/client/cli/ArgumentUtility.java

```diff
@@ -79,7 +88,14 @@ public static void exceptionMessage(Exception exception, String message, int exi
         if (!"".equals(message)) {
             err(message);
         }
-        err(ExceptionUtils.getStackTrace(exception));
+
+        if (LOG.isInfoEnabled()) {
+            err(ExceptionUtils.getStackTrace(exception));
+        } else if (!"".equals(exception.getMessage())) {
+            err(exception.getMessage());
+        } else if ("".equals(message)) {
+            err("An unknown error has occurred. Consider rerunning with --debug");
```

Comment:
Should I be advising users to rerun with `--debug`?

Also, I will change `--debug` to the `DEBUG` constant once https://github.com/dockstore/dockstore-cli/pull/221 is merged.

Reply:
> Should I be advising users to rerun with `--debug`?

That make sense, although we don't have too many `LOG.debug` statements. But we have enough.